File: examples/assets/input_output/code_examples/javascript/simple_calculator.js
// Simple Calculator - JavaScript/Node.js Version
// This calculator adds two numbers from stdin

const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
    terminal: false
});

const lines = [];

rl.on('line', (line) => {
    lines.push(line.trim());
    if (lines.length === 2) {
        const a = parseInt(lines[0]);
        const b = parseInt(lines[1]);
        console.log(a + b);
        rl.close();
    }
});

File: examples/assets/input_output/code_examples/java/SimpleCalculator.java
// Simple Calculator - Java Version
// This calculator adds two numbers from stdin

import java.util.Scanner;

public class SimpleCalculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        System.out.println(a + b);
        scanner.close();
    }
}
